### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by chadyuu**

### CODE REVIEW

**Comment by daviddalpiaz** [comment]
Wow! This is awesome! Thanks for this! Crazy that these weren't caught over the years.

I'm going to merge this but I might revert a few of the small changes afterwords. In the future, I think it'd be slightly more efficient (at least for me as the maintainer) for this to be more like one PR per chapter.

**Comment by chadyuu** [comment]
Thank you for your confirmation.

>I might revert a few of the small changes afterwards.

Please feel free to add comments, and I can revert changes.

> I think it'd be slightly more efficient (at least for me as the maintainer) for this to be more like one PR per chapter.

It makes sense to me. Please let me follow it hereafter.

File: anova.Rmd [hunk 1]
```diff
@@ -171,15 +171,15 @@ where the mean of each group is given by
 
 Here $\alpha_i$ measures the effect of group $i$. It is the difference between the overall mean and the mean of group $i$.
 
-Essentially, the assumptions here are the same as the two sample case, however now, we simply have more groups.
+Essentially, the assumptions here are the same as the two sample cases, however now, we simply have more groups.
```
Comment (daviddalpiaz) [inline anova.Rmd]: Are we actually referencing multiple two sample cases here? If not, I think singular is still correct.

(I'm also now realizing two sample should be two-sample.)

File: anova.Rmd [hunk 2]
```diff
@@ -690,7 +690,7 @@ levels(rats$poison)
 levels(rats$treat)
 ```
 
-Here, 48 rats were randomly assigned both one of three poisons and one of four possible treatments. The experimenters then measures their survival time in tens of hours. A total of 12 groups, each with 4 replicates.
+Here, 48 rats were randomly assigned both one of three poisons and one of four possible treatments. The experimenters then measure their survival time in tens of hours. A total of 12 groups, each with 4 replicates.
```
Comment (daviddalpiaz) [inline anova.Rmd]: I think should be "measured" to be consistent with the past tense from the previous sentence.

File: diagnostics.Rmd [hunk 3]
```diff
@@ -296,7 +296,7 @@ hist(resid(fit_3),
      breaks = 20)
 ```
 
-Above are histograms for each of the three regression we have been considering. Notice that the first, for `fit_1` appears very normal. The third, for `fit_3`, appears to be very non-normal. However `fit_2` is not as clear. It does have a rough bell shape, however, it also has a very sharp peak. For this reason we will usually use more powerful tools such as **Q-Q plots** and the **Shapiro-Wilk test** for assessing the normality of errors.
+Above are histograms for each of the three regressions we have been considering. Notice that the first, for `fit_1` appears very normal. The third, for `fit_3`, appears to be very non-normal. However `fit_2` is not as clear. It does have a rough bell shape, however, it also has a very sharp peak. For this reason we will usually use more powerful tools such as **Q-Q plots** and the **Shapiro-Wilk test** for assessing the normality of errors.
```
Comment (daviddalpiaz) [inline diagnostics.Rmd]: If it works in context, how about "regression models"?

File: model-building.Rmd [hunk 4]
```diff
@@ -276,11 +276,11 @@ Suppose instead of the manufacturer who would like to build a car, we are a cons
...
+The trouble with using RMSE to identify how well a model fits data is that RMSE is **always** (equal or) lower for a larger model. This would suggest that we should always use the largest model possible when looking for a model that predicts well. The problem with this is the potential to **overfit** to the data. So, we want a model that fits well, but does not overfit. To understand overfitting, we need to think about applying a model to seen and unseen data.
 
 Suppose we fit a model using all data available and we evaluate RMSE on this fitted model and all of the seen data. We will call this data the **training** data, and this RMSE the **train** RMSE.
 
-Now, suppose we magically encounter some additional data. To truly asses how well the model predicts, we should evaluate how well our models predicts the response of this data. We will call this data the  **test** data and this RMSE the **test** RMSE.
+Now, suppose we magically encounter some additional data. To truly assess how well the model predicts, we should evaluate how well our model predicts the response of this data. We will call this data the  **test** data and this RMSE the **test** RMSE.
```
Comment (daviddalpiaz) [inline model-building.Rmd:283]: 🤦‍♂️ (Not the first time I've made that typo.)

File: selection.Rmd [hunk 5]
```diff
@@ -14,15 +14,15 @@ After reading this chapter you will be able to:
...
+This suggests that we need a quality criterion that takes into account the size of the model, since our preference is for small models that still fit well. We are willing to sacrifice a small amount of "goodness-of-fit" for obtaining a smaller model. (Here we use "goodness-of-fit" to simply mean how far the data is from the model, the smaller the errors the better. Often in statistics, goodness-of-fit can have a more precise meaning.) We will look at three criteria that do this explicitly: $\text{AIC}$, $\text{BIC}$, and Adjusted $R^2$. We will also look at one, Cross-Validated $\text{RMSE}$, which implicitly considers the size of the model.
```
Comment (daviddalpiaz) [inline selection.Rmd:25]: TIL: Criteria is is the plural of criterion. 🤦‍♂️

File: selection.Rmd [hunk 6]
```diff
@@ -239,9 +239,9 @@ lines(xplot, predict(fit_big_removed, newdata = data.frame(x = xplot)),
       col = "darkorange", lwd = 2, lty = 2)
 ```
 
-We see that on average, the solid blue line for the quadratic model has similar errors as before. It has changed very slightly. However, the dashed orange line for the large model, has a huge error at the point that was removed and is much different that the previous fit.
+We see that on average, the solid blue line for the quadratic model has similar errors as before. It has changed very slightly. However, the dashed orange line for the large model, has a huge error at the point that was removed and is much different from the previous fit.
```
Comment (daviddalpiaz) [inline selection.Rmd]: If it is OK grammatically, I think I prefer "than" which I think is what I tried for, but made a typo. I'm OK with "from" though.

File: selection.Rmd [hunk 7]
```diff
@@ -638,25 +638,25 @@ Notice this is a somewhat "large" model, which uses `r length(coef(autompg_mod_b
...
 
 Note that *linear* models are rather interpretable to begin with. Later in your data analysis careers, you will see more complicated models that may fit data better, but are much harder, if not impossible to interpret. These models aren't very useful for explaining a relationship.
 
-To find small and interpretable models, we would use selection criterion that *explicitly* penalize larger models, such as AIC and BIC. In this case we still obtained a somewhat large model, but much smaller than the model we used to start the selection process.
+To find small and interpretable models, we would use selection criteria that *explicitly* penalize larger models, such as AIC and BIC. In this case we still obtained a somewhat large model, but much smaller than the model we used to start the selection process.
```
Comment (daviddalpiaz) [inline selection.Rmd:647]: Well at least I was consistent in how I got it wrong! :)

**Review by daviddalpiaz — CHANGES_REQUESTED** [review]
Left a few comments. Again, thanks so much for this! What a thorough reading of the text!

File: anova.Rmd (hunk 1 above)
Comment (chadyuu) [reply anova.Rmd]: I updated the content, so please check this commit: https://github.com/daviddalpiaz/appliedstats/pull/128/commits/8e1a0b0eb97c3a11485eccbd2181d05b3a56b385

File: anova.Rmd (hunk 2 above)
Comment (chadyuu) [reply anova.Rmd]: I updated the content, so please check this commit: https://github.com/daviddalpiaz/appliedstats/pull/128/commits/8e1a0b0eb97c3a11485eccbd2181d05b3a56b385

File: diagnostics.Rmd (hunk 3 above)
Comment (chadyuu) [reply diagnostics.Rmd]: I updated the content, so please check this commit: https://github.com/daviddalpiaz/appliedstats/pull/128/commits/0cdf94ebae0025c31b0c81363e01027728d590c6

File: selection.Rmd (hunk 6 above)
Comment (chadyuu) [reply selection.Rmd]: I updated the content, so please check this commit: https://github.com/daviddalpiaz/appliedstats/pull/128/commits/abfbf1800fe76f411501f530b10df2d6473c319f

**Review by daviddalpiaz — APPROVED** [review]

**Comment by daviddalpiaz** [comment]
Merged! Thanks! 🎉

**Pull request merged**